### PR TITLE
R1 reduce jargon & repitition

### PR DESCRIPTION
**Pull request by Aariq**
Addresses #277 (partly) & #278 
In order to reduce repetition and jargon in the use cases sections, we've added a paragraph to the introduction [L__ -L__] explaining a basic git & GitHub workflow and pointing the reader to other resources for learning these tools.  This allows us to focus less on explaining the specific mechanisms of git/Github in the use cases sections and more on the _usefulness_ of the tools. In regards to the specific suggestion about deprioritizing GitHub communication tools, we believe these are extremely useful alternatives to reply-all email chains for a variety of reasons and have attempted to improve the clarity of this section of our manuscript [end of project management section].

1. Adds a paragraph to the intro about basic git/GitHub workflow
2. Removes GitHub jargon from use cases
3. Removes specifics about static website generators
4. Removes sentences on Zenhub and Slack integration in Project managment
5. Adds some cross-refrences
6. Adds a little more detail to collaborative coding section (oops, maybe out of scope for this PR?)

### CODE REVIEW

**Comment by Aariq** [comment]
I didn't do any of the rearranging and merging of sections in this PR because that would make it difficult to track the more minor edits.  Will do that separately after this is reviewed and merged.

File: content/02.introduction.md [hunk 1]
```diff
@@ -19,17 +19,25 @@ This workflow provides a strong and clear advantage over sending files back-and-
...
 
+Version control involves tracking the state of the files in a folder, called a "repository" (see [Box 1](#definitions)), through the evolution of a project.
+A basic workflow using git and GitHub is to 1) make changes to files in a repository on your computer, 2) record related changes in files as a "commit" (see [Box 1](#definitions)) along with a message describing the changes, 3) synchronize commits with GitHub (see "push" in [Box 1](#definitions)) so that the repository on the web is up to date. 
+Collaborators (or you on a different computer) can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).
+Commits act like snapshots and one can revisit the state of the project to any commit to view or undo changes.
+If the modified files are plain text (e.g. computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.
```
Comment (fhillemann) [inline content/02.introduction.md:30]: add a comma and "to": If the modified files are plain text (e.g. computer code or .txt files), only the differences from the previous commit are recorded, allowing you to frequently commit ...

File: content/02.introduction.md [hunk 2]
```diff
@@ -19,17 +19,25 @@ This workflow provides a strong and clear advantage over sending files back-and-
...
+This provides a safe and less cluttered alternative to frequently making full copies of documents at different points in their evolution (e.g. analysis.R, analasys_v2.R, analysis_FINAL.R).
+More technical details about the use of Git and GitHub are beyond the scope of this paper (but see ref, ref, ref).
```
Comment (fhillemann) [inline content/02.introduction.md:32]: could cite [doi.org/10.1371/journal.pcbi.1004947](https://doi.org/10.1371/journal.pcbi.1004947) and [doi.org/10.1371/journal.pcbi.1004668](https://doi.org/10.1371/journal.pcbi.1004668)

File: content/03.use-cases-a.md [hunk 3]
```diff
@@ -10,14 +10,15 @@ A centralized research compendium stored in a version-controlled repository has
 The ability to make a GitHub repository private is particularly advantageous for storing materials without necessarily sacrificing privacy, which may be preferred or even essential for certain compendia. 
 Researchers can then grant specific collaborators read and/or write access to private repositories to pursue pre-publication analyses or writing in privacy, or even to collaborate on research that will permanently remain private.
 
+<!-- Is this paragraph too complicated/jargony?-->
```
Comment (fhillemann) [inline content/03.use-cases-a.md:13]: I think it's fine

File: content/03.use-cases-a.md [hunk 4]
```diff
@@ -50,9 +52,11 @@ GitHub supports a broad set of mechanisms for hosting educational materials.
...
+<!-- remove/simplify stuff about tools to make documents.  out of scope -->
+Syllabi, presentations, and other course materials can be version-controlled and stored in GitHub. 
+Although plain text formats such as markdown or HTML work best with GitHub and help to keep the total size of the repository within limits, any file type can be stored on and downloaded from GitHub including PowerPoint and Word documents.
+Content can then be delivered by sharing a link to the GitHub repository or creating a template repository with assignments that students can use.
+Using GitHub to host a course website is also an option for making course content more widely and easily accessible (see [Hosting a Website](#hosting-a-website)).
+
+Using GitHub to collect student submissions is perhaps the most challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.
```
Comment (fhillemann) [inline content/03.use-cases-a.md:61]: maybe change "the most challenging component" to "a more challenging component"

File: content/04.use-cases-b.md [hunk 5]
```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
...
 
 <!--*Contributors to this section: Kaitlyn Gaynor, Rob Crystal-Ornelas, Ali, Allison Binley*-->
 
-Researchers can asynchronously communicate and individually or collaboratively work in GitHub.
-By forking, branching, and cloning repositories ([Box 1](#definitions)), researchers can simultaneously work on different issues of the same research project to then later merge changes to the main branch project with pull requests.
-Explicit project organization and increased communication in GitHub Issues, Discussions, or within pull requests can help with project development and with potential merge conflicts due to users simultaneously working on the same sections [@doi:10.1007/s10664-019-09774-x].
-Moreover, version control [@https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html#storing-and-archiving-version-controlled-data] and commit reversal features allow researchers to track and progress without worrying about irreparably modifying someone else's work.
+<!-- I think I accidentally added more jargon, but I feel like this section is really important to get right. I'd be open to leaving all the GitHub jargon out and simply citing a source on this instead.  -->
```
Comment (fhillemann) [inline content/04.use-cases-b.md:44]: great work, Eric! In the first sentence, I would change "share in-progress code" to "share in-progress work", to remove the focus on code. Similarly, in Line 52, maybe say "with the click of a button, greatly facilitating peer review" or add an "for example"

**Review by fhillemann — COMMENTED** [review]
thanks, Eric, great work!

File: content/03.use-cases-a.md (hunk 3 above)
Comment (helenweierbach) [reply content/03.use-cases-a.md:13]: I think the first sentence is hard to follow, but could easily be remedied by changing the sentence structure a bit e.g. "Additionally, users have the ability to make their repository private, allowing researchers to store materials without sacrificing privacy which may be preferred, or essential for certain subject matter"

File: content/04.use-cases-b.md [hunk 6]
```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
...
+
+In academic settings, GitHub can also facilitate interactions between research advisors and advisees, or among collaborators, providing a platform for everyone to share in-progress code, and flag specific challenges or questions for their supervisors, mentors, or collaborators ([Table 2](#tbl:roles)).
+Periodic code review [@doi:10.1145/3341525.3387370] can also help identify errors early in the scientific process, and inform further training and mentorship to fill gaps in skills.
+This is facilitated by a group of core features of git and GitHub that allow contributors to work simultaneously on the same project without interferring with eachother's work.
+"Branching" and "forking" (see [Box 1](#definitions)) allow users to work independently on a linked copy of a repository and then later merge their changes into the main project. 
+To signal that changes are ready to be merged, one makes a "pull request" (see [Box 1](#definitions)).
```
Comment (helenweierbach) [inline content/04.use-cases-b.md:50]: I like this section and think it highlights one of the major benefits of using GitHub in scientific research. One thing that may be helpful to touch on, however, is the frequency of when you want to be creating pull requests (or in other words what is the link between branching/forking and pull requests?). Something like "After a team member reaches a new milestone, or addresses issues in code that is ready to be merged one can make a "pull request"... Not sure if it's worth it but we could also add a sentence that discusses something such as "GitHub facilitates team members to frequently branch, commit, and merge code so that large projects avoid spending large amounts of time making software environments, functions, and other portions of their code compatible with other team members when products need to be finished for publication" This would also help to address some of the reviewers comments about how investing in time on the front end can avoid long term issues when code needs to be synchronized for final products. Let me know what you think

File: content/02.introduction.md (hunk 2 above)
Comment (helenweierbach) [reply content/02.introduction.md:32]: perhaps this one as well: Kalliamvakou, Eirini, et al. "Open source-style collaborative development practices in commercial projects using github." 2015 IEEE/ACM 37th IEEE international conference on software engineering. Vol. 1. IEEE, 2015.

**Review by helenweierbach — COMMENTED** [review]
Overall, I think these sections have improved a lot, thanks for all the hard work, Eric!

**Comment by Aariq** [comment]
Thanks for the comments all.  I'll try to incorporate these, merge this PR, then start a new one for rearranging an merging sections (#277)

**Comment by robcrystalornelas** [comment]
Hi @Aariq 👋🏻 , just took a read thru and made a few quick edits in my own branch to the intro and merged here into this PR for your review.

File: content/02.introduction.md [hunk 7]
```diff
@@ -2,34 +2,41 @@
...
 Although the understanding of basic concepts of Git (such as commit, push, pull, checkout; see [Box 1](#definitions)) is necessary, the GitHub web-based platform and its integrated development environments (such as GitHub Desktop) allow users to perform most repository and data management operations without opening Git command-line sessions, making these functionalities available to users who are less familiar with software development.
 
+Version control involves tracking the state of the files in a folder, called a "repository" (see [Box 1](#definitions)), through the evolution of a project.
+A basic workflow using git and GitHub is to 1) make changes to files in a repository on your computer, 2) record related changes in files as a "commit" (see [Box 1](#definitions)) along with a message describing the changes, 3) synchronize commits with GitHub (see "push" in [Box 1](#definitions)) so that the repository on the web is up to date. 
+Collaborators (or you on a different computer) can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).
```
Comment (robcrystalornelas) [inline content/02.introduction.md:27]:
```suggestion
Collaborators can then view your changes online and synchronize them to their own computers (see "pull" in [Box 1](#definitions)).
```

I don't think we need the "or you" part

File: content/02.introduction.md [hunk 8]
```diff
@@ -2,34 +2,41 @@
...
+Commits act like snapshots and one can revisit the state of the project to any commit to view or undo changes.
```
Comment (robcrystalornelas) [inline content/02.introduction.md:28]:
```suggestion
Commits act like snapshots and one can view or even revert the state of the project to any previous commit.
```

File: content/02.introduction.md [hunk 9]
```diff
@@ -2,34 +2,41 @@
...
+If the modified files are plain text (e.g. computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.
```
Comment (robcrystalornelas) [inline content/02.introduction.md:29]:
```suggestion
If the modified files are plain text (_e.g._, computer code or .txt files) only the differences from the previous commit are recorded, allowing you frequently commit changes without causing the size of the project to grow excessively.
```

File: content/02.introduction.md [hunk 10]
```diff
@@ -2,34 +2,41 @@
...
+This provides a safe and less cluttered alternative to frequently making full copies of documents at different points in their evolution (e.g. analysis.R, analasys_v2.R, analysis_FINAL.R).
```
Comment (robcrystalornelas) [inline content/02.introduction.md:30]:
```suggestion
This provides a safe and less cluttered alternative to frequently making full copies of documents at different points in their evolution (_e.g.,_ analysis.R, analasys_v2.R, analysis_FINAL.R).
```

File: content/02.introduction.md [hunk 11]
```diff
@@ -2,34 +2,41 @@
...
+More technical details about the use of Git and GitHub are beyond the scope of this paper (but see ref, ref, ref).
```
Comment (robcrystalornelas) [inline content/02.introduction.md:31]:
```suggestion
More technical details about the use of Git and GitHub are beyond the scope of this paper (but see [@doi:10.1371/journal.pcbi.1004947; @doi:10.1371/journal.pcbi.1004668]).
```

Adding in citations

File: content/03.use-cases-a.md [hunk 12]
```diff
@@ -10,14 +10,15 @@ A centralized research compendium stored in a version-controlled repository has
 The ability to make a GitHub repository private is particularly advantageous for storing materials without necessarily sacrificing privacy, which may be preferred or even essential for certain compendia. 
 Researchers can then grant specific collaborators read and/or write access to private repositories to pursue pre-publication analyses or writing in privacy, or even to collaborate on research that will permanently remain private.
 
+<!-- Is this paragraph too complicated/jargony?-->
 GitHub limits committed file sizes to 100 Mb (megabytes) [@url:https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github], which can make it challenging for centralizing research compendia containing larger file sizes.
```
Comment (robcrystalornelas) [inline content/03.use-cases-a.md:14]:
```suggestion
GitHub limits committed file sizes to 100 Mb (megabytes) [@url:https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github], which can make it challenging to use as a repository for _all_ research files. 
```

Removing jargon

File: content/03.use-cases-a.md [hunk 13]
```diff
@@ -10,14 +10,15 @@ A centralized research compendium stored in a version-controlled repository has
 The ability to make a GitHub repository private is particularly advantageous for storing materials without necessarily sacrificing privacy, which may be preferred or even essential for certain compendia. 
 Researchers can then grant specific collaborators read and/or write access to private repositories to pursue pre-publication analyses or writing in privacy, or even to collaborate on research that will permanently remain private.
 
+<!-- Is this paragraph too complicated/jargony?-->
 GitHub limits committed file sizes to 100 Mb (megabytes) [@url:https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github], which can make it challenging for centralizing research compendia containing larger file sizes. 
 Users may still version large files using Git Large File Storage (<https://git-lfs.github.com>) text pointers, but may have to rely on external file storage alternatives (such as local or cloud-hosting).
```
Comment (robcrystalornelas) [inline content/03.use-cases-a.md:15]:
```suggestion
Users may still version large files using Git Large File Storage (<https://git-lfs.github.com>), but may have to rely on external file storage alternatives (such as local or cloud-hosting).
```

File: content/03.use-cases-a.md [hunk 14]
```diff
@@ -30,17 +31,18 @@ GitHub issues (see Project management section below) can be used to prioritize l
...
+However, one can opt to receive new issues, discussions, and responses as emails and can post replies by email as well.  
+This allows for centralized communication for a team even when some members prefer to use email for communication.
 Unlike emails and messages which can get lost as more new tasks arise, GitHub issues are intentionally closed by repository administrators hiding the issue from view (closed issues remain accessible but not immediately visible).
-Project management in GitHub can also be integrated with third-party applications, such as Zenhub (<https://www.zenhub.com>) or Slack (<https://slack.github.com>).
-Zenhub allows for the enhanced visualization and organization of repositories and their issues, while the GitHub for Slack integration allows notifications from GitHub events to be sent directly to users or group channels [see @url:https://slack.com/help/articles/232289568-GitHub-for-Slack].
+
+<!-- ERS: I removed the Zenhub / Slack section as it seemed like it might fall into R1's definition of "too nerdy". Plus, new GitHub projects has basically all the features of Zenhub-->
```
Comment (robcrystalornelas) [inline content/03.use-cases-a.md:45]: Sounds fine to me.

File: content/03.use-cases-a.md (hunk 4 above)
Comment (robcrystalornelas) [inline content/03.use-cases-a.md:61]:
```suggestion
Using GitHub to collect student submissions is perhaps a more challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.
```

Comment (robcrystalornelas) [reply content/03.use-cases-a.md:61]: I added @fhillemann's suggestion in review comment below

File: content/04.use-cases-b.md [hunk 15]
```diff
@@ -1,17 +1,17 @@
...
-The resulting files from static websites can then be hosted in repositories, from which one can activate GitHub's Pages (<https://pages.github.com>) feature, allowing for the direct live hosting of HyperText Markup Language (HTML), Cascading Style Sheets (CSS) and JavaScript files within a free github.io domain.
+Despite researchers in ecology and evolution generally lacking experience in building or hosting webpages, many tools have been developed to help this process. 
+Simple websites can consist of a single HTML or markdown document that will be rendered as a website by activating GitHub's Pages (<https://pages.github.com>) feature. 
+More complex static websites can now be easily built from a series of markdown documents using a variety of software including literate programming tools like Quarto (<https://quarto.org>) and R packages such as `distill` (<https://github.com/rstudio/distill>) and `blogdown` (<https://bookdown.org/yihui/blogdown>).
```
Comment (robcrystalornelas) [inline content/04.use-cases-b.md:9]: Maybe remove sentence since:
1) want to cut down on length
2) blogdown _is_ more technical and prob out of scope for this paper

File: content/04.use-cases-b.md [hunk 16]
```diff
@@ -1,17 +1,17 @@
...
+The resulting files from static websites can then be hosted in repositories, from which one can activate GitHub's Pages (<https://pages.github.com>) feature to render fully featured and styled websites.
```
Comment (robcrystalornelas) [inline content/04.use-cases-b.md:10]: If we remove previous sentence remove this as well.

File: content/04.use-cases-b.md [hunk 17]
```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
...
-Explicit project organization and increased communication in GitHub Issues, Discussions, or within pull requests can help with project development and with potential merge conflicts due to users simultaneously working on the same sections [@doi:10.1007/s10664-019-09774-x].
-Moreover, version control [@https://sortee-github-hackathon.github.io/manuscript/v/latest/index.html#storing-and-archiving-version-controlled-data] and commit reversal features allow researchers to track and progress without worrying about irreparably modifying someone else's work.
+<!-- I think I accidentally added more jargon, but I feel like this section is really important to get right. I'd be open to leaving all the GitHub jargon out and simply citing a source on this instead.  -->
+
+In academic settings, GitHub can also facilitate interactions between research advisors and advisees, or among collaborators, providing a platform for everyone to share in-progress code, and flag specific challenges or questions for their supervisors, mentors, or collaborators ([Table 2](#tbl:roles)).
```
Comment (robcrystalornelas) [inline content/04.use-cases-b.md:46]: Making more concise. 

```suggestion
In academic settings, GitHub can serve as a platform for everyone (advisor, advisees, collaborators) to share in-progress code, and flag specific challenges or questions for each other ([Table 2](#tbl:roles)).
```

File: content/04.use-cases-b.md [hunk 18]
```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
...
+Periodic code review [@doi:10.1145/3341525.3387370] can also help identify errors early in the scientific process, and inform further training and mentorship to fill gaps in skills.
```
Comment (robcrystalornelas) [inline content/04.use-cases-b.md:47]:
```suggestion
Periodic code review [@doi:10.1145/3341525.3387370] is useful for identifying errors early in the scientific process, and informing further training and mentorship to fill gaps in skills.
```

File: content/04.use-cases-b.md [hunk 19]
```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
...
+This is facilitated by a group of core features of git and GitHub that allow contributors to work simultaneously on the same project without interferring with eachother's work.
```
Comment (robcrystalornelas) [inline content/04.use-cases-b.md:48]:
```suggestion
This is facilitated by a group of core features of git and GitHub ("branching" and "forking" [Box 1](#definitions)) that allow contributors to work simultaneously on the same project without interfering with each other's work.
```

File: content/04.use-cases-b.md [hunk 20]
```diff
@@ -37,17 +37,22 @@ For example, Creative Commons (CC; <https://creativecommons.org/licenses/>) lice
...
+"Branching" and "forking" (see [Box 1](#definitions)) allow users to work independently on a linked copy of a repository and then later merge their changes into the main project.
```
Comment (robcrystalornelas) [inline content/04.use-cases-b.md:49]: Removed this sentence becuase I incorporated content into previous sentence
```suggestion
```

File: content/04.use-cases-b.md (hunk 6 above)
Comment (robcrystalornelas) [reply content/04.use-cases-b.md:50]: I don't know, we mention many frequent commits in previous sentence. And I think it's fine to just talk generally within this paragraph about the different git commands like branch, fork, pull without going into "best practices". If we want to include those could put something in the definition table.

**Review by robcrystalornelas — APPROVED** [review]
Made a few comments and suggested changes throughout. Otherwise LGTM!

File: content/03.use-cases-a.md (hunk 4 above)
Comment (Aariq) [reply content/03.use-cases-a.md:61]: I think I'll go with something like "a more advanced use".

File: content/04.use-cases-b.md (hunk 6 above)
Comment (Aariq) [reply content/04.use-cases-b.md:50]: I see what you're saying @helenweierbach.  I think our readers would probably find some general guidelines useful (like PR == new milestone), but I'm reluctant to make specific recommendations since I think this stuff depends a lot on the group culture.  How about this?
```suggestion
When a set of related changes contributing some new feature or milestone are ready to be evaluated and merged, one makes a "pull request" (see [Box 1](#definitions)).
```

File: content/04.use-cases-b.md (hunk 20 above)
Comment (Aariq) [reply content/04.use-cases-b.md:47]: I'm not seeing the edit to the previous sentence, but I like where this is going.

**Comment by Aariq** [comment]
Ok, I think this is ready to be merged and then I'll work on combining sections (as discussed in #277 and also here: https://hackmd.io/ICM9eNqkQFyLBH2_9G3i_Q?both) in a separate PR
